NetKet/Graph/graph.hpp

```diff
@@ -70,6 +68,8 @@ class Graph : public AbstractGraph {
     return g_->Distances();
   }
 
+  virtual ColorMap EdgeColors() const override { return g_->EdgeColors(); }
```

Comment:
Why this method is virtual?

Reply:
This was an oversight on my part and I've removed the virtual keyword.